### PR TITLE
Improve feedback ui and fix varius bugs

### PR DESCRIPTION
**Pull request by Ell1ott**

### CODE REVIEW

File: src/routes/workspace/sidebar/quickActions/QuickTabs.svelte [hunk 1]
```diff
@@ -50,7 +45,7 @@
 			<ClipboardList size="27"></ClipboardList>
 		</QuickTab>
 	{/if}
-	{#if $currentFile instanceof AssignmentAnswer || ($currentFile instanceof Assignment && $currentFile.isPublic && $answer)}
+	{#if showAssignmentTabs}
```
Comment (arctixdev) [inline src/routes/workspace/sidebar/quickActions/QuickTabs.svelte:48]: The feedback tab should only be visible if the teacher is currently looking at an assignmentAnswer. i.e. $answer

**Review by arctixdev — CHANGES_REQUESTED** [review]

**Comment by arctixdev** [comment]
Looks good otherwise!

**Comment by arctixdev** [comment]
Afk at the moment, but just thinking, does it still show when viewing answers (as a student)?

**Comment by Ell1ott** [comment]
No, it does not. Thanks, good catch 👍

**Review by arctixdev — APPROVED** [review]
Ok, I think it works correctly now. Maybe you could look through my changes, feel free to merge if you approve my changes.

**Comment by arctixdev** [comment]
Shit, what a mess of merges :smile: I think everything still works, u fine with merging now?

**Comment by arctixdev** [comment]
Just gonna merge it now, looks like you are ofline

**Pull request merged**